### PR TITLE
Make RedeyeTool Class Gobject-style constructed

### PR DESCRIPTION
**Pull request by ZenitsuDev**
Make redeyetool gobject-style constructed. Also replaced specified type with `var` when there is clear indication of the instance.

### CODE REVIEW

File: src/editing_tools/RedEyeTool.vala [hunk 1]
```diff
@@ -70,8 +70,11 @@ public class EditingTools.RedeyeTool : EditingTool {
                 RedeyeInstance.MIN_RADIUS, RedeyeInstance.MAX_RADIUS, 1.0);
 
         public RedeyeToolWindow (Gtk.Window container) {
-            base (container);
+            Object (transient_for: container);
```
Comment (ryonakano) [inline src/editing_tools/RedEyeTool.vala:74]: Can you remove this extra new line while we're here?

Comment (ZenitsuDev) [reply src/editing_tools/RedEyeTool.vala:74]: Oh I did not notice. I will. Thanks

File: src/editing_tools/RedEyeTool.vala [hunk 2]
```diff
@@ -45,9 +45,9 @@ public struct EditingTools.RedeyeInstance {
     }
 
     public static RedeyeInstance from_bounds_rect (Gdk.Rectangle rect) {
-        Gdk.Rectangle in_rect = rect;
+        var in_rect = rect;
```
Comment (ryonakano) [inline src/editing_tools/RedEyeTool.vala:48]: Hmm, it looks like we may actually not need the variable `in_rect`; it's essentially the same with the argument `rect` and only an  unnecessary copy happens heere. Maybe we can remove this and replace the occurance of `in_rect` to `rect` in  this method while we're here.

**Review by ryonakano — COMMENTED** [review]
Just one more nitpicking comment, otherwise looking good.

**Comment by ZenitsuDev** [comment]
I'll try that. thanks

**Comment by ZenitsuDev** [comment]
Should be fine by now. Thanks again @ryonakano

**Review by ryonakano — APPROVED** [review]
Looks good, thank you!

**Pull request merged**